src/components/Button/index.tsx

```diff
@@ -0,0 +1,134 @@
+import type { HTMLAttributes, ReactElement } from 'react'
+import { hexToCSSFilter } from 'hex-to-css-filter'
+import styled from '@emotion/styled'
+import type { StyledProps } from '@types'
+import type { Theme } from '@emotion/react'
+
+export const BUTTON_STYLE_KEYS = {
+  GHOST: 'ghost',
+  OUTLINE: 'outline',
+  OUTLINE_DISABLED: 'outlineDisabled',
+  SOLID_DISABLED: 'solidDisabled',
+  SOLID_PRIMARY: 'solidPrimary',
+  SOLID_SUB: 'solidSub'
+}
```

Comment:
```suggestion
} as const
```
객체 단언 시키는게 좋을 것 같습니다!
`BUTTON_STYLE`로 네이밍 변경 후, `applyFontColor()`의 중복 코드 제거에 활용해보시는 건 어떠신가요?